app/assets/v2/js/pages/bounty_detail/web3_modal.js

```diff
@@ -11,17 +11,21 @@ const payWithWeb3 = (fulfillment_id, fulfiller_address, vm, modal) => {
   }
 
   if (token_name == 'ETH') {
-    web3.eth.sendTransaction(
-      {
-        to: fulfiller_address,
-        from: selectedAccount,
-        value: web3.utils.toWei(String(amount)),
-        gasPrice: web3.utils.toHex(5 * Math.pow(10, 9)),
-        gas: web3.utils.toHex(318730),
```

Comment:
From here. We can remove this params, default is 90000 and it will return unused gas.